scripts/lite/shelley-testnet.sh

```diff
@@ -40,27 +62,31 @@ for i in 1 2 3; do
   # - KES signing key
   # - VRF signing key
   # - Operational certificate
+  # VRF keys have already been generated in the shelley genesis create command
 
-  # Generate VRF keys
-  mkdir -p "${data_dir}/genesis/vrf-keys"
-  cardano-cli shelley node key-gen-VRF \
-      --verification-key-file "${data_dir}/genesis/vrf-keys/vrf-$i.vkey" \
-      --signing-key-file      "${data_dir}/genesis/vrf-keys/vrf-$i.skey"
 
   # Generate a KES keys
-  mkdir -p "${data_dir}/genesis/kes-keys"
+  mkdir -p "${data_dir}/node-$i"
   cardano-cli shelley node key-gen-KES \
-    --verification-key-file "${data_dir}/genesis/kes-keys/kes-$i.vkey" \
-    --signing-key-file      "${data_dir}/genesis/kes-keys/kes-$i.skey"
+    --verification-key-file "${data_dir}/node-$i/kes.vkey" \
+    --signing-key-file      "${data_dir}/node-$i/kes.skey"
+
+  # Move genesis delegate keys generated in shelley genesis create command to it's
+  # respective node folder.
+
+  mv "${data_dir}/genesis/delegate-keys/delegate$i.skey"      "${data_dir}/node-$i/hotkey.skey"
+  mv "${data_dir}/genesis/delegate-keys/delegate$i.vkey"      "${data_dir}/node-$i/hotkey.vkey"
+  mv "${data_dir}/genesis/delegate-keys/delegate$i.counter"   "${data_dir}/node-$i/counterFile"
```

Comment:
Should we keep the `.counter` extension for consistency?  Like `hotkey.counter`?